extensions/schema-compare/src/utils.ts

```diff
@@ -83,7 +83,7 @@ function connectionInfoToConnectionProfile(details: azdata.ConnectionInfo): azda
 	};
 }
 
-export async function verifyConnectionAndGetOwnerUri(endpoint: mssql.SchemaCompareEndpointInfo, caller: string): Promise<string> {
+export async function verifyConnectionAndGetOwnerUri(endpoint: mssql.SchemaCompareEndpointInfo, caller: string): Promise<string | undefined> {
```

Comment:
if you are adding undefined please make sure here ever this is called we check for undefined before using...

Reply:
The method calling this is already checking it, not sure why I didn't update this method when I worked on it before :-| .